test/ResponseHeaderMiddlewareFactoryTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DotTest\ResponseHeader;
+
+use Dot\ResponseHeader\Factory\ResponseHeaderMiddlewareFactory;
+use PHPUnit\Framework\MockObject\Exception;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Psr\Container\ContainerExceptionInterface;
+use Psr\Container\ContainerInterface;
+use Psr\Container\NotFoundExceptionInterface;
+
+class ResponseHeaderMiddlewareFactoryTest extends TestCase
+{
+    private ResponseHeaderMiddlewareFactory $responseHeaderMiddlewareFactory;
+
+    private ContainerInterface|MockObject $containerInterface;
+
+    /**
+     * @throws Exception
+     */
+    public function setUp(): void
+    {
+        $this->responseHeaderMiddlewareFactory = new ResponseHeaderMiddlewareFactory();
+        $this->containerInterface              = $this->createMock(ContainerInterface::class);
```

Comment:
Both of these properties seem to be unused - please remove them (including the property declarations).